compute_endpoint/globus_compute_endpoint/strategies/base.py

```diff
@@ -40,60 +43,49 @@ class BaseStrategy:
     from a duplicate logger being added by the thread.
     """
 
-    def __init__(self, *args, threshold: int = 20, interval: float = 5.0):
+    def __init__(self, *args, interval: float = 5.0):
         """Initialize the flowcontrol object.
 
         We start the timer thread here
-        Parameters
-        ----------
-        - threshold (int) : Tasks after which the callback is triggered
-        - interval (int) : seconds after which timer expires
 
+        :param interval: how often (in seconds) to invoke self.strategy
         """
-        self.interchange = None
-        self.threshold = threshold
+        self.interchange: t.Optional[GlobusComputeEngine] = None
```

Comment:
I think we ought to rename this variable to `self.engine`

Reply:
I had thought so as well&nbsp;&mdash;&nbsp;had even done that&nbsp;&mdash;&nbsp;but Mypy educated me: the key part of this change is continuity with the subclasses.  In particular, `kube_simple.py` references `interchange`, and it expects an `Interchange` not an `Engine`.  In other words, there's more work to go here that a simple name change; I was hoping to make this a simple topical change.

Let's absolutely flesh this out later, but that is perhaps ticketed work rather than a "oh, I saw this, and that's ten minutes to just do it" work.